attributes/default.rb

```diff
@@ -20,3 +20,12 @@
 default['rundeck_bridge']['options']['partial-search'] = 'true'
 #<> Default username attribute for the node element in the generated resource xml
 default['rundeck_bridge']['options']['username'] = 'rundeck'
+
+
+# Service setup (Using 'poise')
+default['rundeck_bridge']['poise_service']['options'] = {
```

Comment:
Yes
* it allows a wrapper to specify extra options for a specific Init provider
* I need it in the `recipes/service.rb` to overwrite the `cmd` for the right provider selected by Poise according the OS